source/extensions/filters/network/redis_proxy/conn_pool_impl.cc

```diff
@@ -122,6 +138,91 @@ InstanceImpl::ThreadLocalPool::makeRequest(const std::string& key,
     client->redis_client_->addConnectionCallbacks(*client);
   }
 
+  // cache host for quick lookup via address for redirection support
```

Comment:
nit: please try to take a pass through the PR and start comments with capitals, end with period, use proper grammar, etc.

Reply:
ok